libs/api/domains/cms/src/lib/environments/environment.ts

```diff
@@ -9,6 +9,7 @@ export default {
     'news',
     'page',
   ],
+  nestedContentTypes: ['processEntry'],
```

Comment:
We should probably provide a complete list of types here to fully complete this issue